olive/evaluator/olive_evaluator.py

```diff
@@ -698,9 +698,10 @@ def _inference(
         targets = torch.cat(targets, dim=0)
         logits = torch.cat(logits, dim=0)
         # move model to cpu
-        # don't want model to be kept on gpu since model persists and takes up gpu memory
         if device:
             session.to("cpu")
+        # only move to cpu cannot release gpu memory, call cuda.empty_cache() to release gpu memory
+        torch.cuda.is_available() and torch.cuda.empty_cache()
```

Comment:
better to use if clause to make the logic clear.

Reply:
Updated.